java/src/main/java/org/extism/sdk/manifest/ManifestHttpRequest.java

```diff
@@ -3,6 +3,12 @@
 import java.util.Map;
 
 // FIXME remove this and related stuff if not supported in java-sdk
-public record ManifestHttpRequest(String url, Map<String, String> header, String method) {
+public class ManifestHttpRequest {
```

Comment:
I'd make the fields final, create accessor methods and create a constructor for passing the required parameters in.